spinnaker_components/filter/filter_common.h

```diff
@@ -0,0 +1,150 @@
+/* Common filter operator code
```

Comment:
The contents of this doesn't look super-common i.e. ``dma_tag_ops`` and ``_transform_page``. Can it be moved into the c-file - It's not unreasonably large